pephub/routers/api/v1/namespace.py

```diff
@@ -419,13 +434,28 @@ async def remove_from_stars(
         )
 
 
-@api.get(
-    "/namespace/info",
+@namespaces.get(
+    "/info",
     summary="Get information list of biggest namespaces",
-    tags=["namespace"],
+    response_model=ListOfNamespaceInfo,
 )
 async def get_namespace_information(
     limit: Optional[int] = DEFAULT_LIMIT_INFO,
     agent: PEPDatabaseAgent = Depends(get_db),
 ) -> ListOfNamespaceInfo:
     return agent.namespace.info(limit=limit)
+
+
+@namespaces.get(
+    "/projects_list",
+    summary="Get list of annotations for list of projects",
+    response_model=AnnotationList,
+)
+async def get_namespace_projects_list(
+    namespace_access: List[str] = Depends(get_namespace_access_list),
+    agent: PEPDatabaseAgent = Depends(get_db),
+    registry_paths: Annotated[List[str], Query()] = None,
+):
+    return agent.annotation.get_by_rp_list(
+        registry_paths=registry_paths, admin=namespace_access
+    )
```

Comment:
I'm going to propose that we implement this functionality on the `/projects` endpoint that exists already and allow a query param that lets us specify which projects we want to fetch specifically. Some reasoning for this is:
- this current approach isn't very [restful](https://www.ibm.com/topics/rest-apis) and 
- it duplicates already existing functionality.

I like how [spotify handles this](https://developer.spotify.com/documentation/web-api/reference/get-several-tracks)
Its also similar to [GitHubs implementation](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories) where you can specify filters to get specific subsets of a users repos.

Reply:
I think we can keep the current `/projects` endpoint as it stands and allow an optional query param `registries` that defaults to `None` if `None`: get all projects, otherwise only get the subsetted list.